File: Examples/DotNET/SDK/CSharp/Fields/InsertPageNumberField.cs
using System;
using Aspose.Cloud;

namespace CSharp.Fields
{
    class InsertPageNumberField
    {
        public static void Run()
        {            
            string inputfile = "doc-sample.doc";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);
                       
            // Insert Page Number Field into the Document
            PageNumberSettings pageNumberSettings = new PageNumberSettings("{PAGE} of {NUMPAGES}", "right", true, true);
            Common.WordsService.InsertDocumentPageNumbers(inputfile, Common.GetOutputFilePath(inputfile, true), Common.FOLDER, pageNumberSettings);

            string dataDir = Common.DownloadFile(inputfile, string.Empty);

            Console.WriteLine("\nPage number inserted successfully.\nFile saved at " + dataDir);
        }
    }
}

File: Examples/DotNET/SDK/VisualBasic/Fields/InsertPageNumberField.vb
Imports Aspose.Cloud

Namespace Fields
    Class InsertPageNumberField
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Insert Page Number Field into the Document
            Dim pageNumberSettings As New PageNumberSettings("{PAGE} of {NUMPAGES}", "right", True, True)
            Common.WordsService.InsertDocumentPageNumbers(inputfile, Common.GetOutputFilePath(inputfile, True), Common.FOLDER, pageNumberSettings)

            Dim dataDir As String = Common.DownloadFile(inputfile, String.Empty)

            Console.WriteLine(Convert.ToString(vbLf & "Page number inserted successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace